frontend/src/components/Game/Game.js

```diff
@@ -125,13 +127,25 @@ export default function Game(props)
         ev.target.innerHTML = data;
     }
 
+    if(time === 11){                 // if time is 180, it will automatically click submit answer
+                                      // You can change it if you want user to have more time to play
+        document.getElementById("submit").click();
```

Comment:
if the intention is to use jQuery for this .click, it can be done in pure HTML actions https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/click